src/components/FieldArray/FieldArray.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { node, string } from 'prop-types';
+import classNames from 'classnames';
+import { FieldArray } from 'react-final-form-arrays';
+
+import css from './FieldArray.css';
+
+const FieldArrayComponent = props => {
+  const { rootClassName, className, id, name, children } = props;
+
+  const classes = classNames(rootClassName || css.root, className);
+
+  return (
+    <FieldArray id={id} name={name} className={classes}>
```

Comment:
Sorry, but I don't think it makes much sense for this to exists as a separate component (FieldArray wrapper for FieldArray). There are no styles attached to this component (and the styles are likely to be different in different use scenarios when `import { FieldArray } from 'react-final-form-arrays';` is used. 

It is also good to note that every time FieldArrays are used inside Final Form, the form needs to pass `arrayMutators` as props to the FinalForm component itself. So, wrapping the usage of `react-final-form-arrays` library with separate component like this one might be a bit confusing since it doesn't work out of the box.